Generator.Equals.Runtime/Generator.Equals.Runtime.csproj

```diff
@@ -4,6 +4,7 @@
         <TargetFramework>netstandard2.0</TargetFramework>
         <RootNamespace>Generator.Equals</RootNamespace>
         <IsPackable>true</IsPackable>
+        <LangVersion>8.0</LangVersion>
```

Comment:
I don't quite understand your concerns. I am using the latest version of C# even in net4.0 and it works well. Some features like Range operators will only work in net core/net5/net6, but this is limited by the compiler

Reply:
It's just that I see another simple improvement that could improve the code - the file-scope namespace, which requires C# 10.